File: Iniciante/1041 - Coordenadas de um Ponto/C#/1041 - Coordenadas de um Ponto.cs
using System;

class URI {
    static void Main() {
        // separa os valores para cada posição do array
        string[] coordenadas = Console.ReadLine().Split(' ');
            float p1 = float.Parse(coordenadas[0]);
            float p2 = float.Parse(coordenadas[1]);

        if(p1>0 && p2>0) { // saída com condicional
            Console.WriteLine("Q1");

        } else if(p1<0 && p2>0) {
            Console.WriteLine("Q2");

        } else if(p1<0 && p2<0) {
            Console.WriteLine("Q3");

        } else if(p1>0 && p2<0) {
            Console.WriteLine("Q4");

        } else if(p1==0 && p2==0) {
            Console.WriteLine("Origem");

        } else if(p1==0 && p2!=0) {
            Console.WriteLine("Eixo Y");

        } else if(p1!=0 && p2==0) {
            Console.WriteLine("Eixo X");
        }
    }
}

File: Iniciante/1041 - Coordenadas de um Ponto/Python/1041 - Coordenadas de um Ponto.py
# recebe os valores na mesma linha
x, y = input().split(' ')

# converte os valores para float
x = float(x)
y = float(y)

# identifica a posição do ponto
if x > 0 and y > 0:
    print('Q1')

elif x < 0 and y > 0:
    print('Q2')

elif x < 0 and y < 0:
    print('Q3')

elif x > 0 and y < 0:
    print('Q4')

elif x == 0 and y == 0:
    print('Origem')

elif x == 0 and y != 0:
    print('Eixo Y')

elif x != 0 and y == 0:
    print('Eixo X')